tests/spread/plugins/godeps/legacy-build-run/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Build and run a basic Godeps snap with no base
+
+environment:
+  SNAP_DIR: ../snaps/godeps
+  GOBIN/gobin: gobin
+  GOBIN/no_gobin:
+
+systems: [ubuntu-*]
+
+prepare: |
+  # Make sure no additional snaps are installed before running this test.
+  $TOOLS_DIR/restore.sh
```

Comment:
This is part of the global restore step, is there a reason it needs to be done again here?